department-of-veterans-affairs/abd-vro
# Mock Schema Registry

## What was the problem?
There's no mock schema registry in place to conduct local testing using avro schema serialization/deserialization 

Associated tickets or Slack threads:
- #1902 

## How does this fix it?[^1]
A new schema registry image, `confluentinc/cp-schema-registry`, has been added to serve as a mock schema registry. It will automatically register five BIE contention classification schemas once the service is up.

## How to test this PR
run e2e integration test


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
